nvm/pmemobj/tuple.py

```diff
@@ -0,0 +1,96 @@
+import collections
+import sys
+
+from .compat import recursive_repr, abc
+from .list import PersistentList
+from _pmem import ffi
+
+TUPLE_POBJPTR_ARRAY_TYPE_NUM = 50
```

Comment:
Right, I hardcoded it originally with the intent of "fixing it later".  The python hash function is not guaranteed to be portable, even though for the most part it is.  One possibility is to use a hashlib function on the type name (like I did to get a portable hash in the dict implementation).